doc/source/glossary.rst

```diff
@@ -55,7 +62,7 @@ Map, mapping
     attempts to use mathematical functional notation where possible.  Within volatility a mapping is most often
     used to refer to the function for translating addresses from a higher layer (domain) to a lower layer (range).
     For further information, please see
-    `Function (mathematics) in wikipedia https://en.wikipedia.org/wiki/Function_(mathematics)`
+    `https://en.wikipedia.org/wiki/Function_(mathematics)`.
```

Comment:
I think this should be `[Function (mathematics) in wikipedia](https://en.wikipedia.org/wiki/Function_(mathematics))`, although I'm not sure how that'll work with brackets in the URL...